<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QDialog" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>811</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReset">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>570</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>570</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNotify">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>761</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>You win!</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
